crates/uv-distribution/src/source/built_wheel_metadata.rs

```diff
@@ -29,8 +29,8 @@ pub(crate) struct BuiltWheelMetadata {
 impl BuiltWheelMetadata {
     /// Find a compatible wheel in the cache.
     pub(crate) fn find_in_cache(tags: &Tags, cache_shard: &CacheShard) -> Option<Self> {
-        for directory in files(cache_shard) {
```

Comment:
This is unrelated change, but I got a bit confused here between directory/files while reading the code.